### DISCUSSION TITLE
How should I do this? (Customizing the console route)

### DISCUSSION THREAD

**Discussion by eduardolucioac** [okd-project/okd#744] [Q&A]
In this documentation [Customizing the console route](https://docs.okd.io/latest/web_console/customizing-the-web-console.html#customizing-the-console-route_customizing-web-console) there is how I should change the "Web Console" settings for new routes. However, **one thing I didn't understand** is **how I can create/edit this configuration?** Let me explain better... What kind of parameter should I pass to "oc"? Something like this?

EXAMPLE

```
oc edit service/downloads -n openshift-console
```

MODEL

```
oc edit <KIND>/<RSRC_NAME> -n <NAME_SPACE> 
```

In other words, the instruction "Set the custom hostname and, if needed, set TLS on the ingress config's spec" doesn't make it clear to me whether I should edit or create a certain configuration and where I should do it.

Could you give me an example? That would help a lot!

**Thanks! =D**
